SWDevelopment/TestScripts/ServoTestScript

```diff
@@ -0,0 +1,28 @@
+  public static void main(String[] args)
+  {
+    PCA9685 servoBoard = new PCA9685();   // 0x40 is the default address
+    servoBoard.setPWMFreq(60); // Set frequency to 60 Hz
+    int servoMin = 150;   // Min pulse length out of 4096
+    int servoMax = 600;   // Max pulse length out of 4096
+
+    final int CONTINUOUS_SERVO_CHANNEL = 14;
```

Comment:
Do we know how these channels layout on the PWM controller?  So when we run this test we know which one to plug it into?

Reply:
I would assume we have to change the CONTINUOUS_SERVO_CHANNEL value to whatever the channel is for the new controller, right? Also, isn't the servoMin & servoMax supposed to be a double between -1 and 1?

Reply:
servoMin and servoMax are the actual PWM frequencies I assume, aka Hz for each 3.3 V -> 5V ->3.3 V transformation